### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by adam-palazzo**
This updates the README to the orbitdb repo standards, changed dates on licenses

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
-# orbit-bot
+# <Orbit Bot>
```
Comment (RichardLitt) [inline README.md]: Remove brackets.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+
+[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https://gitter.im/orbitdb/Lobby)
+[![npm version](https://badge.fury.io/js/<orbit-bot>.svg)](https://www.npmjs.com/package/<orbit-bot>)
```
Comment (RichardLitt) [inline README.md]: The brackets in `<Replace>` are also meant to removed.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
+[![node](https://img.shields.io/node/v/<orbit-bot>.svg)](https://www.npmjs.com/package/<orbit-bot>)
 
 > Bot for Orbit communication network that caches messages
 
-**WIP**
```
Comment (RichardLitt) [inline README.md:5]: Keep this here. Maybe add:

`Status: **WIP**`.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,28 @@
...
+## Install
+
+This project uses [npm](http://npmjs.com/) and [nodejs](https://nodejs.org/).
+
+```sh
+npm i <orbit-bot>
```
Comment (RichardLitt) [inline README.md]: Again, no brackets. And is this on NPM?

File: README.md [hunk 5]
```diff
@@ -1,13 +1,28 @@
...
+```
 
 ## Contribute
 
-Open an issue! This is a WIP.
+This is a WIP but feel free to submit a PR! You can reach us [on Gitter](https://gitter.im/orbitdb/Lobby), or in the [issues section](https://github.com/orbitdb/<orbit-bot>/issues).
```
Comment (RichardLitt) [inline README.md]: Say open an issue before opening a PR. WIP means more hand-holding for contribs.

Comment (RichardLitt) [reply README.md]: Remove brackets.

File: README.md [hunk 6]
```diff
@@ -1,13 +1,28 @@
...
+
+We also have **regular community calls**, which we announce in the issues in [the @orbitdb welcome repository](https://github.com/orbitdb/welcome/issues). Join us!
+
+
+For specific guidelines for contributing to this repository, check out the [Contributing guide](CONTRIBUTING.md). For more on contributing to OrbitDB in general, take a look at the [@OrbitDB welcome repository](https://github.com/orbitdb/welcome). Please note that all interactions in [@OrbitDB](https://github.com/orbitdb) fall under our [Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## License
 
-[MIT](LICENSE) © 2016-2018 Protocol Labs Inc., Haja Networks Oy
+[MIT](LICENSE) © 2016-2019 Haja Networks Oy
```
Comment (RichardLitt) [inline README.md]: This needs to reflect the license file.

**Review by RichardLitt — CHANGES_REQUESTED** [review]
Close.

**Comment by RichardLitt** [comment]
Also, update the Code of Conduct. This version is old.

File: README.md [hunk 7]
```diff
@@ -1,13 +1,29 @@
-# orbit-bot
+# Orbit Bot
```
Comment (RichardLitt) [inline README.md]: I'd prefer to keep the name as is, in general.

File: README.md [hunk 8]
```diff
@@ -1,13 +1,29 @@
...
+
+[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https://gitter.im/orbitdb/Lobby)
+[![node](https://img.shields.io/node/v/orbit-bot.svg)](https://www.npmjs.com/package/orbit-bot)
```
Comment (RichardLitt) [inline README.md]: The link doesn't match the badge? What is the node badge doing?

File: README.md [hunk 9]
```diff
@@ -1,13 +1,29 @@
...
 
 > Bot for Orbit communication network that caches messages
 
-**WIP**
+## Status: WIP
```
Comment (RichardLitt) [inline README.md]: Nope: Just:

```md

Status: **WIP**

```

File: README.md [hunk 10]
```diff
@@ -1,13 +1,29 @@
...
+
+## Install
+
+This project [nodejs](https://nodejs.org/).
```
Comment (RichardLitt) [inline README.md]: Uses?

File: LICENSE [hunk 11]
```diff
@@ -1,7 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2016-2018 Protocol Labs Inc.
-Copyright (c) 2018 Haja Networks Oy
+Copyright (c) 2018-2019 Haja Networks Oy
```
Comment (RichardLitt) [inline LICENSE]: Per new company policy, stop updating years and don't touch them anymore.

File: README.md [hunk 12]
```diff
@@ -1,13 +1,29 @@
...
+```sh
+git clone https://github.com/orbitdb/orbit-bot
+```
 
 ## Contribute
 
-Open an issue! This is a WIP.
+This is a WIP but feel free to submit an issue! You can reach us [on Gitter](https://gitter.im/orbitdb/Lobby), or in the [issues section](https://github.com/orbitdb/orbit-bot/issues).
+
+We also have **regular community calls**, which we announce in the issues in [the @orbitdb welcome repository](https://github.com/orbitdb/welcome/issues). Join us!
```
Comment (RichardLitt) [inline README.md:21]: Change (everywhere) from @orbitdb to @OrbitDB. Seems nicer.

Comment (RichardLitt) [reply README.md:21]: Whoa! GitHub changes it to lowercase. Uh. Scratch that, I guess? Fascinating. Maybe we should stop doing this everywhere.

File: README.md [hunk 13]
```diff
@@ -1,13 +1,29 @@
...
+
+
+For specific guidelines for contributing to this repository, check out the [Contributing guide](CONTRIBUTING.md). For more on contributing to OrbitDB in general, take a look at the [@OrbitDB welcome repository](https://github.com/orbitdb/welcome). Please note that all interactions in [@OrbitDB](https://github.com/orbitdb) fall under our [Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## License
 
-[MIT](LICENSE) © 2016-2018 Protocol Labs Inc., Haja Networks Oy
+[MIT](LICENSE) © 2016-2018 Protocol Labs Inc., © 2018-2019 Haja Networks Oy
```
Comment (RichardLitt) [inline README.md]: As above, don't change the year. Also, don't add another copyright symbol - one is enough.

File: package.json [hunk 14]
```diff
@@ -12,14 +12,19 @@
...
   "homepage": "https://github.com/orbitdb/orbit-bot",
   "bugs": "https://github.com/orbitdb/orbit-bot/issues",
   "repository": {
     "type": "git",
     "url": "https://github.com/orbitdb/orbit-bot.git"
   },
   "contributors": [
-    "Haadcode <haad@ipfs.io>"
+    "Haadcode <haad@ipfs.io>",
+    "adam-palazzo"
```
Comment (RichardLitt) [inline package.json:27]: Run `git authors`. Is there another contributor we should add here? (Note: then remove Authors, don't commit the file that is created in the repo.)

**Review by RichardLitt — CHANGES_REQUESTED** [review]
Thanks Adam.

File: README.md (hunk 12 above)
Comment (adam-palazzo) [reply README.md:21]: I wondered why this kept popping up, I swore I was triple checking every time :+1:

File: CODE_OF_CONDUCT.md [hunk 15]
```diff
@@ -57,6 +57,65 @@ further defined and clarified by project maintainers.
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported by contacting the project team at haad@ipfs.io. All
 complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is# Contributor Covenant Code of Conduct
```
Comment (RichardLitt) [inline CODE_OF_CONDUCT.md]: What's going on here?

File: README.md (hunk 10 above)
Comment (RichardLitt) [reply README.md]: Still needs uses.

File: package.json [hunk 16]
```diff
@@ -6,20 +6,25 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "Haad",
```
Comment (RichardLitt) [inline package.json]: Keep.

**Review by RichardLitt — CHANGES_REQUESTED** [review]
Smaller changes.

File: README.md (hunk 10 above)
Comment (adam-palazzo) [reply README.md]: Done!

File: CODE_OF_CONDUCT.md (hunk 15 above)
Comment (adam-palazzo) [reply CODE_OF_CONDUCT.md]: Not sure, fixed it though

Comment (RichardLitt) [reply CODE_OF_CONDUCT.md]: Still doubled.

Comment (adam-palazzo) [reply CODE_OF_CONDUCT.md]: I see what was going on. Actually fixed it

**Comment by RichardLitt** [comment]
Danke.

**Pull request merged**